frontend/components/sections/FAQsSection.tsx

```diff
@@ -53,67 +53,52 @@ export function FAQsSection({
                   )}
                </Box>
                <Accordion defaultIndex={[0]} allowMultiple>
-                  {faqs.map((faq, index) => {
-                     return (
-                        <FAQAccordionItem key={index}>
-                           {({ isExpanded }) => {
-                              return (
-                                 <>
-                                    <h3>
-                                       <AccordionButton p="4">
-                                          <Box
-                                             flex="1"
-                                             textAlign="left"
-                                             fontWeight="medium"
-                                             color={
-                                                isExpanded
-                                                   ? 'brand.500'
-                                                   : 'gray.900'
-                                             }
-                                          >
-                                             {faq.question}
-                                          </Box>
-                                          <FaIcon
-                                             icon={
-                                                isExpanded
-                                                   ? faMinusCircle
-                                                   : faPlusCircle
-                                             }
-                                             color="gray.400"
-                                          />
-                                       </AccordionButton>
-                                    </h3>
-                                    <AccordionPanel
-                                       pb="5"
-                                       color="gray.700"
-                                       bg="white"
-                                    >
-                                       {faq.answer}
-                                    </AccordionPanel>
-                                 </>
-                              );
-                           }}
-                        </FAQAccordionItem>
-                     );
-                  })}
+                  {faqs.map((faq, index) => FAQAccordionItem({ faq, index }))}
```

Comment:
I don't remember if this syntax is better than using jsx? I think `<FAQAccordionItem key={index} .../>` is a bit better cause it puts the "key" prop right here in the loop, rather than having to pass it into the function.

Reply:
I think @danielbeardsley is right, using components as functions is definitely not a conventional approach in React

Reply:
Technically, this is no different than a typical map function (which is just a function that returns a component). The main difference is that the function body is outside of the scope and thus can't access any other local vars.

Reply:
Thanks for pointing this out, we rushed this one out the door. I'll open a fix PR 👍🏻

Reply:
https://github.com/iFixit/react-commerce/pull/1922